geomesa-blobstore/geomesa-blobstore-core/src/main/java/org/locationtech/geomesa/blob/core/impl/AccumuloGeoMesaBlobStore.java

```diff
@@ -0,0 +1,129 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+
+package org.locationtech.geomesa.blob.core.impl;
+
+import org.geotools.data.Query;
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore;
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStoreFactory;
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStoreParams;
+import org.locationtech.geomesa.blob.core.AccumuloBlobStore;
+import org.locationtech.geomesa.blob.core.interop.GeoMesaBlobStore;
+import org.opengis.filter.Filter;
+
+import java.io.File;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class AccumuloGeoMesaBlobStore implements GeoMesaBlobStore {
+
+    protected AccumuloBlobStore accumuloBlobStore;
+
+    public AccumuloGeoMesaBlobStore(Map<String, Serializable> dataStoreParams) throws Exception {
+        AccumuloDataStoreFactory accumuloDataStoreFactory = new AccumuloDataStoreFactory();
+        AccumuloDataStore ds = (AccumuloDataStore) accumuloDataStoreFactory.createDataStore(dataStoreParams);
+        if (ds == null) {
+            throw new Exception("Error initializing AccumuloGeoMesaBlobStore");
```

Comment:
@elahrvivaz any opinion on making this an illegal argument exception and removing ```Exception``` from the signature? @AndrewAnnex picky comment - maybe change comment to "Unable to locate Accumulo DataStore with given parameters" instead of just can't initialize since it'll tell us that via the stacktrace